tests/ui/issues/issue-52049.stderr

```diff
@@ -2,12 +2,10 @@ error[E0716]: temporary value dropped while borrowed
   --> $DIR/issue-52049.rs:6:10
    |
 LL |     foo(&unpromotable(5u32));
-   |     -----^^^^^^^^^^^^^^^^^^-
+   |     -----^^^^^^^^^^^^^^^^^^-- temporary value is freed at the end of this statement
```

Comment:
Why?

Reply:
Because I changed the RPO of
```
  A
 / \
B C
\ /
 D
```
from ` A C B D ` to `A B C D`.

I can pull that change out, if you'd prefer.